### ISSUE TITLE
installation missing gem journald-logger

### ISSUE THREAD

**Issue by tomhub** [jbox-web/redmine_git_hosting#783]
I had to add manually into Gemfile `gem 'journald-logger'` and re-run bundle install command to be able to run redmine in puma. Otherwise, puma exists ungracefully:
```Apr 09 08:44:39 sop bundle[2646]: Puma starting in single mode...
Apr 09 08:44:39 sop bundle[2646]: * Version 3.12.6 (ruby 2.7.2-p137), codename: Llamas in Pajamas
Apr 09 08:44:39 sop bundle[2646]: * Min threads: 0, max threads: 16
Apr 09 08:44:39 sop bundle[2646]: * Environment: production
Apr 09 08:44:46 sop bundle[2646]: ! Unable to load application: NameError: uninitialized constant Journald
Apr 09 08:44:46 sop bundle[2646]: bundler: failed to load command: puma (/usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/bin/puma)
Apr 09 08:44:46 sop bundle[2646]: /usr/share/webapps/redmine/plugins/redmine_git_hosting/lib/redmine_git_hosting/journal_logger.rb:3:in `<module:RedmineGitHosting>': uninitialized constant Journald (NameError)
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/plugins/redmine_git_hosting/lib/redmine_git_hosting/journal_logger.rb:1:in `<top (required)>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.2/lib/active_support/dependencies/interlock.rb:14:in `block in loading'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.2/lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/activesupport-5.2.4.2/lib/active_support/dependencies/interlock.rb:13:in `loading'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/engine.rb:478:in `block (2 levels) in eager_load!'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/engine.rb:477:in `each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/engine.rb:477:in `block in eager_load!'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/engine.rb:475:in `each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/engine.rb:475:in `eager_load!'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/engine.rb:356:in `eager_load!'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/application/finisher.rb:69:in `each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/application/finisher.rb:69:in `block in <module:Finisher>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/initializable.rb:32:in `instance_exec'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/initializable.rb:32:in `run'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/initializable.rb:61:in `block in run_initializers'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:228:in `block in tsort_each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:431:in `each_strongly_connected_component_from'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:349:in `block in each_strongly_connected_component'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:347:in `each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:347:in `call'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:347:in `each_strongly_connected_component'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:226:in `tsort_each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/2.7.0/tsort.rb:205:in `tsort_each'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/initializable.rb:60:in `run_initializers'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/railties-5.2.4.2/lib/rails/application.rb:361:in `initialize!'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/config/environment.rb:16:in `<top (required)>'
Apr 09 08:44:46 sop bundle[2646]:         from config.ru:3:in `require'
Apr 09 08:44:46 sop bundle[2646]:         from config.ru:3:in `block in <main>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/builder.rb:116:in `eval'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/builder.rb:116:in `new_from_string'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/builder.rb:105:in `load_file'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/builder.rb:66:in `parse_file'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/lib/puma/configuration.rb:320:in `load_rackup'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/lib/puma/configuration.rb:245:in `app'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/lib/puma/runner.rb:147:in `load_and_bind'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/lib/puma/single.rb:98:in `run'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/lib/puma/launcher.rb:186:in `run'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/lib/puma/cli.rb:80:in `run'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/gems/puma-3.12.6/bin/puma:10:in `<top (required)>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/bin/puma:23:in `load'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/share/webapps/redmine/vendor/bundle/ruby/2.7.0/bin/puma:23:in `<top (required)>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `load'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:63:in `kernel_load'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/cli/exec.rb:28:in `run'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/cli.rb:494:in `exec'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/cli.rb:30:in `dispatch'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/cli.rb:24:in `start'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/exe/bundle:49:in `block in <top (required)>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/lib/ruby/gems/2.7.0/gems/bundler-2.2.15/exe/bundle:37:in `<top (required)>'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/bin/bundle:23:in `load'
Apr 09 08:44:46 sop bundle[2646]:         from /usr/bin/bundle:23:in `<main>'
```

**Comment by alexandermeindl** [comment]
Hi @tomhub 

this is fixed in Master branch, see #778

**Comment by tomhub** [comment]
Thank you, I see. I followed installation, and checked out 5.0.0, where issue is repeated.
